### ISSUE TITLE
TS Server fatal error:  Debug Failure. False expression: y:/myhometv/material/node_modules/@types/aria-query/index.d.ts linked to nonexistent file //192.168.1.55/Abdul/myhometv/material/node_modules/@types/aria-query/index.d.ts

### ISSUE THREAD

**Issue by iqappdev** [microsoft/vscode#182283]
Type: <b>Bug</b>

❗️❗️❗️ Please fill in the sections below to help us diagnose the issue ❗️❗️❗️

**TypeScript Version:** 5.0.4

**Steps to reproduce crash**

1.
2.
3.

**TS Server Log**

❗️ Server logging disabled. To help us fix crashes like this, please enable logging by setting:

```json
"typescript.tsserver.log": "verbose"
```

After enabling this setting, future crash reports will include the server log.

**TS Server Error Stack**

Server: `semantic`

```
Error: Debug Failure. False expression: y:/myhometv/material/node_modules/@types/aria-query/index.d.ts linked to nonexistent file //192.168.1.55/Abdul/myhometv/material/node_modules/@types/aria-query/index.d.ts
    at realPath (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:42011:9)
    at getOriginalAndResolvedFileName (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:40972:28)
    at resolveTypeReferenceDirective (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:41053:45)
    at Object.resolve (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:117475:42)
    at resolveNamesWithLocalCache (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:122896:29)
    at Object.resolveTypeReferenceDirectiveReferences (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:122961:12)
    at InferredProject2.resolveTypeReferenceDirectiveReferences (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:171601:33)
    at resolveTypeReferenceDirectiveNamesWorker (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:118162:20)
    at resolveTypeReferenceDirectiveNamesReusingOldState (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:118332:14)
    at createProgram (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:117949:27)
    at synchronizeHostData (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:135422:15)
    at Object.getProgram (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:135494:5)
    at InferredProject2.updateGraphWorker (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:172176:41)
    at InferredProject2.updateGraph (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:172113:32)
    at _ProjectService.assignOrphanScriptInfoToInferredProject (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:174423:13)
    at _ProjectService.assignProjectToOpenedScriptInfo (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:176043:12)
    at c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:176273:64
    at flatMap (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:2556:17)
    at _ProjectService.applyChangesInOpenFiles (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:176273:24)
    at updateOpen (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:177418:29)
    at c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:179985:69
    at IpcIOSession.executeWithRequestId (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:179977:14)
    at IpcIOSession.executeCommand (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:179985:29)
    at IpcIOSession.onMessage (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:180027:51)
    at process.<anonymous> (c:\Users\IMCR-12\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:181594:14)
    at process.emit (node:events:513:28)
    at emit (node:internal/child_process:958:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:84:21)
```

VS Code version: Code 1.78.2 (b3e4e68a0bc097f0ae7907b217c1119af9e03435, 2023-05-10T14:39:26.248Z)
OS version: Windows_NT x64 10.0.19045
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-4440 CPU @ 3.10GHz (4 x 3093)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: unavailable_off<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: unavailable_off<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|7.94GB (2.25GB free)|
|Process Argv|--crash-reporter-id 30974095-c459-4b1c-9b6e-f3f97c4a791e|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (20)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.3.1
blackbox|Bla|0.10.79
bracket-pair-color-dlw|Bra|0.0.6
es7-react-js-snippets|dsz|4.4.3
gitlens|eam|13.6.0
vscode-html-css|ecm|1.13.1
prettier-vscode|esb|9.12.0
auto-rename-tag|for|0.1.10
html-preview-vscode|geo|0.2.5
todo-tree|Gru|0.0.226
rest-client|hum|0.25.1
vscode-peacock|joh|4.2.2
git-graph|mhu|1.30.0
vscode-docker|ms-|1.25.0
csharp|ms-|1.25.7
indent-rainbow|ode|8.3.1
material-icon-theme|PKi|4.27.0
LiveServer|rit|5.7.9
JavaScriptSnippets|xab|1.8.0
ReactSnippets|xab|2.4.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30294714
vstes263cf:30335440
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
2e4cg342:30602488
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
showlangstatbar:30737416
azdwalk:30721579
pythonms35:30701012
pythonfmttext:30731395
pythoncmv:30731968
fixhidewlkth:30730051

```

</details>

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's covering the same as another one we already have. Thus, we closed this one as a duplicate. You can search for [similar existing issues](https://github.com/microsoft/vscode/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3A%22new%20release%22). See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Happy Coding!
